templates/operations.ts.hbs

```diff
@@ -132,7 +132,7 @@
       {{#each ../parameters}}
         "{{{name}}}": optionParams["{{{name}}}"]
         {{! if it is a common path parameter, fallback to client config for undefined }}
-        {{#if (isCommonPathParameter name)}} !== undefined ? optionParams["{{{name}}}"] : configParams["{{{name}}}"]{{/if}}
+        {{#if (isCommonPathParameter name)}} !== undefined ? optionParams["{{{name}}}"] : configParams?.["{{{name}}}"]{{/if}}
```

Comment:
Why did we require all these changes to use optional chaining in this template?

Reply:
I originally had typescript errors from the generated typescript due to configParams potentially being undefined, but I just regenerated the typescript again and no longer see those issues... Might be something wrong with my environment but reverted those changes